ArcGISRuntimeSDKQt_QMLSamples/Scenes/AnimateImagesWithImageOverlay/AnimateImagesWithImageOverlay.qml

```diff
@@ -0,0 +1,208 @@
+// [WriteFile Name=AnimateImagesWithImageOverlay, Category=Scenes]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Layouts 1.12
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime 100.8
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property var timerIntervals: [17,34,68]
+    readonly property var imageFrameRefreshRate: ["60 fps","30 fps","15 fps"]
+    readonly property url dataPath: System.userHomePath +  "/ArcGIS/Runtime/Data/3D/ImageOverlay/PacificSouthWest"
+
+    // Create new Timer and set the timeout interval to 68
+    // 68 ms interval timer equates to approximatey 15 frames a second
+    Timer {
+        id: timer
+        interval: 68
+        repeat: true
+        property var index: 0
+
+        // connect to the triggered signal to load and display a new image frame each time
+        onTriggered: {
+            // Create an image frame with the url to the image file and a extent
+            let imageFrame = ArcGISRuntimeEnvironment.createObject("ImageFrame", {
+                                                                       url: dataPath + "/" + imageFrameFolder.fileNames()[index],
+                                                                       extent: imageFrameExtent
+                                                                   });
+
+            // set image frame to image overlay
+            imageOverlay.imageFrame = imageFrame;
+
+            // increment the index to keep track of which image to load next
+            index++;
+
+            // reset index once all files have been loaded
+            if (i === imageFrameFolder.fileNames().length) {
```

Comment:
`imageFrameFolder.fileNames().length` could probably be cached to avoid the method call every time